README.md

```diff
@@ -7,9 +7,9 @@ The repository provides a collection of vision language models, benchmarks, and
 
 ## 💡 News
 
+- [2024/10/31] The fine-tuned VILA-M3-13B checkpoint is released on [HuggingFace](https://huggingface.co/MONAI/Llama3-VILA-M3-13B).
```

Comment:
We released three ckpts. Can you add links to all of them
https://huggingface.co/MONAI/Llama3-VILA-M3-3B
https://huggingface.co/MONAI/Llama3-VILA-M3-8B
https://huggingface.co/MONAI/Llama3-VILA-M3-13B

Reply:
Fixed. Thanks!